### PR TITLE
Improve detection alert and general subscription management

### PR DESCRIPTION
**Pull request by lennartkoopmann**
A few bugfixes and improvements for detection alert and general subscription management:

* Actions now also list subscribed detection alerts
* The "Manage" link of detection alert subscriptions moved to the type title instead of it's own "Manage" column (which is standard across the UI)
* Organization administrators had a hard time finding their *Events & Actions* management page because it was nested under their organization management page. This makes sense for super admins, but not for org admins. This PR moves the management link out of the org management page and to it's own sidebar link if the logged in user is an org admin.

### CODE REVIEW

File: src/main/java/app/nzyme/core/events/actions/EventActionUtilities.java [hunk 1]
```diff
@@ -42,6 +44,14 @@ public static EventActionDetailsResponse eventActionEntryToResponse(EventActionE
...
         }
 
+        List<DetectionEventTypeDetailsResponse> detectionEventSubscriptions = Lists.newArrayList();
+        for (DetectionType subscription : subscribedDetectionEvents) {
+            detectionEventSubscriptions.add(DetectionEventTypeDetailsResponse.create(
+                    subscription.name(),
+                    subscription.getTitle(),
+                    subscription.getSubsystem().name()
+            ));
+        }
```
Comment (uskr) [inline src/main/java/app/nzyme/core/events/actions/EventActionUtilities.java:54]: Have seen you using this pattern a few times. You can do this:

`List<DetectionEventTypeDetailsResponse> detectionEventSubscriptions = subscribedDetectionEvent.stream().
                map(o -> DetectionEventTypeDetailsReponse.create(o.name(), o.getTitle(), o.getSubsystem().name()))
                .collect(Collectors.toList());`

No need to change all these, but might save you some verbosity in the future.

**Review by uskr — APPROVED** [review]
Added some comments, but otherwise good to go.

**Comment by uskr** [comment]
I noticed this was not tagged as ready for review. Just ignore the approval and add more code to the PR and i will review it again if thats the case.

**Comment by lennartkoopmann** [comment]
> I noticed this was not tagged as ready for review. Just ignore the approval and add more code to the PR and i will review it again if thats the case.

Sorry, I just forgot the tag. :joy:  Thank you!

File: src/main/java/app/nzyme/core/events/actions/EventActionUtilities.java (hunk 1 above)
Comment (uskr) [reply src/main/java/app/nzyme/core/events/actions/EventActionUtilities.java:54]: for loop perform better and uses less memory for small lists. stream can be parallelized automatically by the vm, so good for large or "endless" datasets.

Comment (lennartkoopmann) [reply src/main/java/app/nzyme/core/events/actions/EventActionUtilities.java:54]: :+1:

**Pull request merged**